.github/workflows/msys2_mingw_w64.yml

```diff
@@ -0,0 +1,71 @@
+name: MSYS2 + Mingw-w64
```

Comment:
This is Github Actions, right? We currently do our Unix testing on Travis and our Windows testing on Appveyor... I wonder if it would be best to add another entry on Appveyor or to enable another CI in this repo :thinking:

Reply:
Right.
In my experience, AppVeyor is slow with many jobs because AppVeyor doesn't run jobs in parallel with free plan.
The latest build https://ci.appveyor.com/project/hishamhm/luarocks/builds/35412465 takes 50min. If we add more jobs to AppVeyor, it will take 1hour. So I choose GitHub Actions.

But I can understand your concern. I can add these jobs to AppVeyor not GitHub Actions if it's better for us.